trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0..10]
+  end
```

Comment:
The name should have a maximum of 10 characters. If it's longer it should trim the word.

- Your trimmer is returning 11 characters. Remember that the range [0..10] represents the indexes starting from 0, therefore 0 represents the first character and 9 represents the 10th character. Your range should be [0..9].